README.md

```diff
@@ -130,14 +130,25 @@ PHP config format is a new [Symfony best practice](https://twitter.com/symfony_e
 
 In the end, it's best to combine few of basic sets and drop [particular rules](/docs/rector_rules_overview.md) that you want to try:
 
-```yaml
-# rector.yaml
-parameters:
-    sets:
-        - code-quality
+```php
+<?php
+// rector.php
+
+declare(strict_types=1);
 
-services:
-    Rector\Php74\Rector\Property\TypedPropertyRector: null
+use Rector\Core\Configuration\Option;
+use Rector\Set\ValueObject\SetList;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Rector\Php74\Rector\Property\TypedPropertyRector;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $parameters = $containerConfigurator->parameters();
+
+    $services = $containerConfigurator->services();
+    $services->set(TypedPropertyRector::class);
```

Comment:
@TomasVotruba  Not sure if the services configuration is done correctly here.

Reply:
I'm not sure what you mean. What is expected?

Reply:
Is it the same as below?
```yaml
services:
    Rector\Php74\Rector\Property\TypedPropertyRector: null
```

Reply:
Yes